### PR TITLE
Image meta data i8

### PR DESCRIPTION
**Pull request by mbredif**
Removal of most of the hardcoded camera parameters and camera urls in the projective texturing module (issue #8)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,54 +0,0 @@
-<!-- SIMPLE API DEMO   -->
```
Comment (tbroyer) [inline index.html:1]: How do you "run" the project without an HTML page? Is it somehow moved into the itowns-demos repo? Color me puzzled…

Comment (tbroyer) [inline index.html:1]: How do you "run" the project without an HTML page? Is it somehow moved into the itowns-demos repo? Color me puzzled…

Comment (vpicavet) [inline index.html:1]: Examples are now inside sample data repos

Comment (tbroyer) [inline index.html:1]: Yet, how do you "run" the project? I mean, the instructions in the readme
are now broken: you need to clone the demos project and open different URLs.
Le 4 févr. 2016 18:37, "Vincent Picavet" notifications@github.com a
écrit :

Comment (vpicavet) [inline index.html:1]: Yes, needs update, working on it

Comment (vpicavet) [inline index.html]: Yes, needs update, working on it

**Pull request merged**